upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1044,13 +1056,6 @@ func ChannelForCluster(c *kops.Cluster) (*kops.Channel, error) {
 	return kops.LoadChannel(channelLocation)
 }
 
-// needsStaticUtils checks if we need our static utils on this OS.
```

Comment:
I made a code comment that we should fix this, rather than having an empty func.

Reply:
OK - can you keep the comment about why it is hard ("(but this has to work with AMI names, and maybe even forked AMIs)")